datadog/datadog_test.go

```diff
@@ -73,7 +73,9 @@ func TestDefaultTextConfig(t *testing.T) {
 func TestNewTextEncoder(t *testing.T) {
 	cfg := DefaultTextConfig()
 	enc := NewTextEncoder()
-	if enc.Config != cfg {
```

Comment:
Why do we need this change?

Reply:
It's what I mentioned in a comment I added above
> Done, by the way adding the func as field in TextEncoder meant that in datadog_test.go a test failed because it was now not possible to compare TextConfigs due to the func field being there.
So I don't know how to really solve it other than that, since we can't != anymore due to the function field because it complains about it, deepEqual doesn't work either and there's no equal operator overriding in Go afaik 😄